axon/layer.go

```diff
@@ -1283,22 +1283,24 @@ func (ly *Layer) SpikeFmG(ni int, nrn *Neuron, ctime *Time) {
 func (ly *Layer) PostAct(ni int, nrn *Neuron, ctime *Time) {
 }
 
-// SendSpike sends spike to receivers -- last step in Cycle, integrated
-// the next time around.
-// Writes to sending projections for this neuron.
-func (ly *Layer) SendSpike(ni int, nrn *Neuron, ctime *Time) {
-	if nrn.Spike == 0 {
-		return
-	}
-	ly.Pools[nrn.SubPool].Inhib.FBsRaw += 1.0 // note: this is immediate..
-	if nrn.SubPool > 0 {
-		ly.Pools[0].Inhib.FBsRaw += 1.0
-	}
-	for _, sp := range ly.SndPrjns {
-		if sp.IsOff() {
+// SendSpike sends spike to receivers for all neurons that spiked
```

Comment:
This seems like it makes perfect sense for Cpp, but in Go there is no easy way to go from the implementation to the interface, because there are not explicitly connected, no? But trying to keep the comment in sync also feels useless and error prone

Reply:
That's a good point about not being easy to get to the interface.
I suppose the comment could just refer readers to the interface then.